src/spicy/spicyz/glue-compiler.cc

```diff
@@ -1475,6 +1507,23 @@ struct VisitorZeekType : hilti::visitor::PreOrder<hilti::Result<hilti::Expressio
             if ( export_.skip )
                 continue;
 
+            // Special-case: Lift up elements of anonymous bitfields.
+            if ( auto bf = f.type.tryAs<hilti::type::Bitfield>(); bf && f.is_anonymous ) {
+                for ( const auto& b : bf->bits() ) {
+                    auto ztype = createZeekType(b.itemType());
+                    if ( ! ztype )
+                        return ztype.error();
+
+                    fields.emplace_back(create_record_field(b.id(), *ztype, f.is_optional, export_.log));
+                }
+
+                continue;
```

Comment:
This `continue` is hidden pretty well. Can we instead make this more explicit, e.g.,

```
if (is_bitfield) {
    ...
}

else if (is_anon) {
    ...
}

else {
    // Existing handling
}
```